Resources/Prototypes/_EE/Entities/NanoTrasen/Modsuits/praetorian.yml

```
@@ -0,0 +1,163 @@
+    - type: EmitsSoundOnMove
+      soundCollection:
+        collection: FootstepHardsuitMedium
+        params:
+          volume: -5
+      requiresWorn: true
+      distanceWalking: 2
+      distanceSprinting: 3
+    - type: SealableClothingVisuals
+      visualLayers:
+        outerClothing:
+        - state: equipped-OUTERCLOTHING-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)